Data/Set.hs

```diff
@@ -13,14 +13,38 @@
 -- Maintainer  :  libraries@haskell.org
 -- Portability :  portable
 --
--- An efficient implementation of sets.
+--
+-- = Finite Sets
+--
+-- The @'Set' e@ type represents a set of elements of type @e@. A 'Set' is
+-- strict in the element type and most operations require that @e@ be an
```

Comment:
Surely you mean strict in its *elements*, not their *type*. I think you should move the strictness to a new sentence after the one about `Ord`.

Reply:
Oops, yup that what I meant.

Reply:
Maybe this should just be a bullet list, it may read better since both the sentences are so short. WDYT?